easybuild/easyconfigs/s/Spark/Spark-2.3.1-CrayGNU-19.03-Hadoop-2.7.eb

```diff
@@ -0,0 +1,86 @@
+easyblock = 'Tarball'
+
+name = 'Spark'
+version = '2.3.1'
+
+javaver = 'jdk1.8.0_51'
+
+versionsuffix = '-Hadoop-2.7'
+
+homepage = 'http://spark.apache.org'
+description = """Spark is Hadoop MapReduce done in memory"""
+
+toolchain = {'name': 'CrayGNU', 'version': '19.03'}
+
+patches = [('spark-2.3.1-Hadoop-2.7.patch', 1)]
+
+sources = ['%(namelower)s-%(version)s-bin-hadoop2.7.tgz']
+source_urls = [
+    'http://apache.belnet.be/%(namelower)s/%(namelower)s-%(version)s/',
+    'http://www.eu.apache.org/dist/%(namelower)s/%(namelower)s-%(version)s/',
+    'http://www.us.apache.org/dist/%(namelower)s/%(namelower)s-%(version)s/',
+]
+
+checksums = ['e87499e5417a64341cbda25e087632dd9f6ce7ad249dfeba47d9d02a51305fc2']
+
+dependencies = [
+    ('java/%s' % javaver, EXTERNAL_MODULE),
```

Comment:
It fails for the missing dependency. Do we ask ops to provide it or are we responsible for this one?